src/BBT.StructureTools/BBT.StructureTools.csproj

```diff
@@ -22,7 +22,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.3">
+    <PackageReference Include="BBT.Maybe" Version="2.0.0" />
+    <PackageReference Include="BBT.StrategyPattern" Version="1.0.0" />
```

Comment:
These dependencies need to be added to the nusepc file as well

Reply:
Fixed